layers/org/packages.el

```diff
@@ -290,6 +290,47 @@ Will work on both org-mode and any mode that accepts plain html."
       (spacemacs/set-leader-keys
         "Cc" 'org-capture)
 
+      (defvar spacemacs--org-hide-docs-meta-tags-t nil)
+
+      (defun spacemacs//org-hide-docs-meta-tags ()
+        "Hide org export tags and annotations."
+        (unless spacemacs--org-hide-docs-meta-tags-t
+          (progn
+            (setq spacemacs--org-hide-docs-meta-tags-t t)
+
+            (defvar spacemacs--org-tag-f-o
+              (face-attribute 'org-tag :foreground))
+            (defvar spacemacs--org-meta-line-f-o
+              (face-attribute 'org-meta-line :foreground))
+            (defvar spacemacs--org-document-info-keyword-f-o
+              (face-attribute 'org-document-info-keyword :foreground))
+
+            (let ((bg (face-attribute 'default :background)))
+              (set-face-attribute 'org-tag                   nil :foreground bg)
+              (set-face-attribute 'org-meta-line             nil :foreground bg)
+              (set-face-attribute 'org-document-info-keyword nil :foreground bg)))))
+
+      (defun spacemacs//org-unhide-docs-meta-tags ()
+        "Unhide org export tags and annotations."
+        (when spacemacs--org-hide-docs-meta-tags-t
+          (progn
+            (setq spacemacs--org-hide-docs-meta-tags-t nil)
+
+            (set-face-attribute 'org-tag                   nil :foreground spacemacs--org-tag-f-o)
+            (set-face-attribute 'org-meta-line             nil :foreground spacemacs--org-meta-line-f-o)
+            (set-face-attribute 'org-document-info-keyword nil :foreground spacemacs--org-document-info-keyword-f-o))))
+
+      (defun spacemacs//org-toggle-docs-meta-tags-hook ()
+        "Make `org-mode' meta tags transparent when viewing README.org
+        files for layers with `view-mode' enabled."
+        (if (and
+             (string-match (concat (expand-file-name "~") "/.emacs.d/layers/.*README\.org")  buffer-file-name)
```

Comment:
@syl20bnr  should I also remove this?  I feel like it might be undesirable behaviour in some cases. For example, when someone reviewing non-Spacemacs related org files :confused: